packages/overlay/test/overlay-trigger.test.ts

```diff
@@ -109,6 +109,14 @@ describe('Overlay Trigger', () => {
         await elementUpdated(testDiv);
     });
 
+    afterEach(async () => {
```

Comment:
Recently, I get caught by the underlying test issue that this ensures has been cleaned up more and more, so I'm very excited to get this into the repo!